spec/support/author.rb

```diff
@@ -1,18 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Support/author was removed from codebase years ago. This spec file should have been removed then.